Generator/Sources/FileRenderer.swift

```diff
@@ -15,43 +15,109 @@
+                if let brief = member.brief?.trimmingCharacters(in: .whitespacesAndNewlines), !brief.isEmpty {
+                    result.append(": \(brief)")
+                }
+            }
         } else {
-            result.append("\n- Examples:")
-            for example in examples {
-                result.append("\n    - `\(example)`")
+            result.append("\n- Type: \(attribute.type)")
+        }
+        if let examples = attribute.examples {
+            if examples.count == 1 {
+                result.append("\n- Example: `\(examples[0])`")
+            } else {
+                result.append("\n- Examples:")
+                for example in examples {
+                    result.append("\n    - `\(example)`")
+                }
             }
         }
+
+        if let note = attribute.note?.trimmingCharacters(in: .whitespacesAndNewlines), !note.isEmpty {
+            result.append("\n\n\(note)")
+        }
+
+        return result.prefixLines(with: "/// ")
     }
 
-    if let note = attribute.note?.trimmingCharacters(in: .whitespacesAndNewlines), !note.isEmpty {
-        result.append("\n\n\(note)")
+    /// Render the `@available` Swift attribute for a deprecated OTel attribute
+    ///
+    /// - Parameters:
+    ///   - deprecated: The `Deprecated` enum that describes the deprecation
+    ///   - extendedTypeName: The name of the type being extended with this attribute (i.e. OTelAttribute, or SpanAttributes). Used in the `renamed` argument, if applicable.
+    /// - Returns: A string representing the `@available` attribute for Swift
+    func renderDeprecatedAttribute(_ deprecated: Deprecated) -> String {
+        var result = "@available(*, deprecated"
+        switch deprecated {
+        case let .renamed(renamed_to, note):
+            if let renamedTo = try? attributeIdToSwiftMemberPath(renamed_to) {
+                result.append(", renamed: \"\(renamedTo)\"")
+            }
+            if let note = note?.trimmingCharacters(in: .whitespacesAndNewlines) {
+                result.append(", message: \"\(note)\"")
+            }
+        case let .obsoleted(note):
+            var message = "Obsoleted"
+            if let note = note?.trimmingCharacters(in: .whitespacesAndNewlines) {
+                message.append(": \(note)")
+            }
+            result.append(", message: \"\(message)\"")
+        case let .uncategorized(note):
+            if let note = note?.trimmingCharacters(in: .whitespacesAndNewlines) {
+                result.append(", message: \"\(note)\"")
+            }
+        }
+        result.append(")")
+        return result
     }
+}
 
-    return result.prefixLines(with: "/// ")
+/// Return the standard Swift member name for an attribute, on the final type that contains the attribute.
+/// Example: "foo.bar_baz" -> "barBaz"
+/// The provided namespace must be the parent of the attribute ID.
+func attributeMemberName(_ attributeId: String, _ namespace: Namespace) throws -> String {
```

Comment:
```suggestion
func attributeMemberName(_ attributeID: String, _ namespace: Namespace) throws -> String {
```

Reply:
Fixed in: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/19/commits/4807300ea3b3a63190a981ad5dc45963d5c33b3e